src/Inkluzitron/Models/Vote/VoteDefinitionParserResult.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Inkluzitron.Models.Vote
+{
+    public struct VoteDefinitionParserResult
+    {
+        public bool Success => ProblemDescription == null;
```

Comment:
1) Přesunul bych to pod ProblemDescription.
2) prázdný řetězec je legit? Pokud ne, tak v tom případě `string.IsNullOrEmpty`

Reply:
1. Přesunuto.
2. V problem description buď je nebo není hodnota, takže takhle je to za mě OK.